testing/xsel/APKBUILD

```diff
@@ -2,18 +2,17 @@
 # Maintainer: dai9ah <dai9ah@protonmail.com>
 pkgname=xsel
 pkgver=1.2.0
-pkgrel=0
+pkgrel=1
 pkgdesc="Command-line program for manipulating the X selection"
 url="http://www.vergenet.net/~conrad/software/xsel"
 arch="all"
 license="MIT"
-makedepends="libxt-dev"
+makedepends="libxt-dev autoconf automake libtool"
 subpackages="$pkgname-doc"
-source="http://www.vergenet.net/~conrad/software/$pkgname/download/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kfish/xsel/archive/$pkgver.tar.gz"
 
 build() {
-	cd "$builddir"
+	./autogen.sh
```

Comment:
please separate the ./autogen.sh into prepare()

Reply:
autogen.sh separated into prepare function.